packages/jest-config/src/index.js

```diff
@@ -17,24 +22,34 @@ import readConfigFileAndSetRootDir from './read_config_file_and_set_root_dir';
 
 function readConfig(
   argv: Argv,
-  packageRoot: string,
+  packageRootOrConfig: Path | InitialOptions,
   // Whether it needs to look into `--config` arg passed to CLI.
   // It only used to read initial config. If the initial config contains
   // `project` property, we don't want to read `--config` value and rather
   // read individual configs for every project.
   skipArgvConfigOption?: boolean,
+  parentConfigPath?: ?Path,
```

Comment:
what does `?` on both sides mean? From my understanding `name: ?type` allows `null`, `undefined` or a value, while `name?: type` allows `undefined` or a value.

https://flow.org/en/docs/types/primitives/#toc-maybe-types

Reply:
I'm not very proficient with Flow. I just added the `?` on the LHS to satisfy the checker. Maybe I can remove the one left?

Reply:
Yeah, I think so 🙂

Reply:
Removing it worked.